stages/org.osbuild.ostree.remotes

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python3
+"""
+Configure OSTree remotes for a repository.
+"""
+
+
+import os
+import sys
+import subprocess
+
+import osbuild.api
+
+
+SCHEMA = """
+"additionalProperties": false,
+"properties": {
+  "repo": {
+    "description": "Location of the OSTree repo.",
+    "type": "string"
+  },
+  "remotes": {
+    "description": "Configure remotes for the system repository",
```

Comment:
Same as before.  Nest `remotes` under `repo`?

Also based on the code, `repo` should be required.

Reply:
`repo` should indeed be required.